### ISSUE TITLE
CSV Import Fail

### ISSUE THREAD

**Issue by sunflowerno0b** [NickvisionApps/Denaro#754]
Hello,

Fresh install, attempted to import CVS and it failed. Checked open and closed issues found the following
https://github.com/NickvisionApps/Denaro/issues/673
https://github.com/NickvisionApps/Denaro/issues/523

Downloaded beta as recommended in above threads: 
https://github.com/NickvisionApps/Denaro/releases/tag/2023.6.0-beta2

But the issue persists.  Also the help page doesn't work on beta2 or stable. 

Heres my details
```org.nickvision.money
2023.11.0
GTK 4.12.3
libadwaita 1.4.0
Flatpak
en-US
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```

While I am here - anyone know of a way to convert CSV to  OFX in the command line or flatpak?

**Comment by nlogozzo** [comment]
Few things: 

> https://github.com/NickvisionApps/Denaro/releases/tag/2023.6.0-beta2

This is a way out of date release. All of the changes/fixes in V2023.6.0-beta2 are inside V2023.11.0 already as that is the newer version.

> Also the help page doesn't work on beta2 or stable.

Docs can be accessed here: https://htmlpreview.github.io/?https://raw.githubusercontent.com/NickvisionApps/Denaro/main/NickvisionMoney.Shared/Docs/html/C/import-export.html

CSV IS NOT A UNIVERSAL FORMAT. 
You must ensure that your CSV file format matches that of what Denaro is looking for, otherwise, it won't work. Read that docs page I linked above for more info.

If you are 100% sure that your CSV file is formated correctly and it still isn't working, please send it here so we can take a look.

**Comment by sunflowerno0b** [comment]
Thank you! I will attempt to build my own ofx & attempt. If not, I will just manually add. ---A.Le 4 déc. 2023 à 20:26, Nick Logozzo ***@***.***> a écrit :﻿
Few things:

https://github.com/NickvisionApps/Denaro/releases/tag/2023.6.0-beta2

This is a way out of date release. All of the changes/fixes in V2023.6.0-beta2 are inside V2023.11.0 already as that is the newer version.

Also the help page doesn't work on beta2 or stable.

Docs can be accessed here: https://htmlpreview.github.io/?https://raw.githubusercontent.com/NickvisionApps/Denaro/main/NickvisionMoney.Shared/Docs/html/C/import-export.html
CSV IS NOT A UNIVERSAL FORMAT.
You must ensure that your CSV file format matches that of what Denaro is looking for, otherwise, it won't work. Read that docs page I linked above for more info.
If you are 100% sure that your CSV file is formated correctly and it still isn't working, please send it here so we can take a look.

**Comment by nlogozzo** [comment]
Like I said if you get the correct formated CSV and still have troubles with it, feel free to reopen this issue.

Furthermore, it may be easier to make a QIF or OFX file with your data and import that since those are universal formats that Denaro will have a better good with. However, they are only limited to importing transactions and groups, not all of the features Denaro supports.

Again everything is explained in detail in the docs :)
